src/utils/referencedBy.js

```diff
@@ -5,14 +5,20 @@
  *
  * @param {IOCozyFile} file - An io.cozy.files document
  * @param {string} referencedBy - An Doctype where document is referenced
+ * @param {string} [id] - An ID of reference
  * @returns {boolean}
  */
-export const isReferencedBy = (file, referencedBy) => {
-  if (file?.relationships?.referenced_by?.data?.length > 0) {
-    const references = file.relationships.referenced_by.data
+export const isReferencedBy = (file, referencedBy, id) => {
+  const references =
+    file?.relationships?.referenced_by?.data || file?.referenced_by || []
+
+  if (!id) {
     return !!references.find(reference => reference.type === referencedBy)
   }
-  return false
+
+  return !!references.find(
```

Comment:
Maybe you can replace `!!references.find(...)` by `references.some(...)`

Reply:
Helpers will be implemented in [cozy-client](https://github.com/cozy/cozy-client/releases/tag/v27.5.1)] now!
So I will delete this file and update cozy-client package in the dependencies.
PS: I made the changes on https://github.com/cozy/cozy-client/pull/1076.